### ISSUE TITLE
Disable error checking?

### ISSUE THREAD

**Issue by birgersp** [Borvik/vscode-postgres#197]
Hi,

I've seen there are a couple of issues created about the error checking. Personally my only gripe with it is that it displays a "Problem" with my query in VSCode, even after I have closed (deleted) the file that contained the error.

I would like to propose a setting that disables the error checking altoghter. I don't really need the error checker in VSCode, reading the returned message from the host is fine for me. So if my query **does** contain an error, I'll see it in the returned message.

In short; Please add the possibility to disable the query error checker.
